src/text/parsers/blob.rs

```diff
@@ -0,0 +1,87 @@
+//! Parsing logic for the text representation of blob values.
+
+use base64::DecodeError;
+use nom::IResult;
+use nom::branch::alt;
+use nom::bytes::streaming::{is_a, tag};
+use nom::character::streaming::{alphanumeric1, char};
+use nom::combinator::{map_res, opt, recognize};
+use nom::multi::many0_count;
+use nom::sequence::{delimited, pair};
+
+use crate::text::parsers::whitespace;
+use crate::text::TextStreamItem;
+
+/// Matches the text representation of a blob value, decodes it, and returns the resulting bytes
+/// as a [TextStreamItem::Blob].
+pub(crate) fn parse_blob(input: &str) -> IResult<&str, TextStreamItem> {
+    map_res::<_, _, _, _, DecodeError, _, _>(
+        delimited(
+            pair(tag("{{"), opt(whitespace)),
+            recognize_base64_data,
+            pair(opt(whitespace), tag("}}")),
```

Comment:
It's not required, but Rust style allows (and even encourages) it because it makes it easy to add and remove items from the list. Any line can be cut/pasted into a different position without needing to rejigger the commas. After this PR is merged, I'll do a crate-wide pass with `rustfmt` to clean up anything it disagrees with.

Reply:
Maybe add a TODO (or Issue) to run rustfmt as part of the CI or commit.